linetools/spectralline.py

```diff
@@ -120,8 +124,18 @@ def cut_spec(self, normalize=False):
             else:
                 sig = sig / self.analy['spec'].conti[pix]
 
-        # Return
-        return fx, sig, wave
+        # Velocity?
+        if relvel:
+            # Calculate
+            self.analy['spec'].velo = self.analy['spec'].relative_vel(
+                self.wrest*(1+self.attrib['z']))
+            # Cut
+            velo = self.analy['spec'].velo[pix] 
+            # Return
+            return fx, sig, velo
+        else:
+            # Return
+            return fx, sig, wave
```

Comment:
I am not a fan of changing the return values based on a keyword flag.  Can you return a dictionary instead? i.e.:

```python
return fx, sig, dict(vel=velo, wave=wave)
```

or 

```python
return dict(fx=fx, sig=sig, vel=velo, wave=wave)
```

(I like the first option better)